vshard/router/init.lua

```diff
@@ -1083,14 +1089,46 @@ local function router_group_map_callrw_args(router, bucket_ids, bucket_args)
     return grouped_args
 end
 
+--
+-- Set the appropriate mode according to bucket_ids option for backward
+-- compatibility (in case of opts_mode is nil) and check the given opts_mode
+-- correctness in other cases.
+--
+local function router_set_map_callrw_mode(opts_mode, bucket_ids)
+    if opts_mode == nil then
+        local mode = bucket_ids and MAP_CALLRW_PARTIAL or MAP_CALLRW_FULL
+        return mode, nil
```

Comment:
You don't need to return trailing `nil`s in Lua. Can be simplified to `return bucket_ids and MAP_CALLRW_PARTIAL or MAP_CALLRW_FULL`.